TLM/TLM/UI/LegacySubToolMixin.cs

```diff
@@ -0,0 +1,72 @@
+﻿namespace TrafficManager.UI {
+    using System;
+    using ColossalFramework.UI;
+    using TrafficManager.U;
+    using UnityEngine;
+
+    [Obsolete("Used in legacy GUI for migration to new subtool base clas")]
+    public class LegacySubToolMixin {
+        [Obsolete("Used in legacy GUI")]
+        internal static GUILayoutOption[] LegacyEmptyOptionsArray = new GUILayoutOption[0];
+
+        [Obsolete("Used in legacy GUI")]
+        protected void LegacyDragWindow(ref Rect window) {
+            Vector2 resolution = UIView.GetAView().GetScreenResolution();
+            window.x = Mathf.Clamp(window.x, 0, UIScaler.MaxWidth - window.width);
+            window.y = Mathf.Clamp(window.y, 0, UIScaler.MaxHeight - window.height);
+
+            bool primaryMouseDown = Input.GetMouseButton(0);
+            if (primaryMouseDown) {
+                GUI.DragWindow();
+            }
+        }
+
+        [Obsolete("Used in legacy GUI")]
+        private Texture2D LegacyWindowTexture {
```

Comment:
do we need to mark even private members as obsolete?

Reply:
is it out of fear that someone might change the scope in future?

Reply:
Visual convenience. Entire class will  be deleted after this branch is complete.